server/src/index.ts

```diff
@@ -0,0 +1,35 @@
+import type Express from 'express';
+import {createSpace as createSpaceImpl, getCookie} from './data.js';
+import {handleRequest as handleRequestImpl} from '../endpoints/handle-request.js';
+import {transact} from './pg.js';
+import type {MutatorDefs} from 'replicache';
+
+export async function spaceExists(spaceID: string) {
```

Comment:
How does this function relate to the one in `handle-space.ts`?

Reply:
redundant. streamlined this as well